deployment/helm/charts/danswer/templates/configmap.yaml

```diff
@@ -7,7 +7,7 @@ metadata:
 data:
   INTERNAL_URL: "http://{{ include "danswer-stack.fullname" . }}-api-service:{{ .Values.api.service.port | default 8080 }}"
   POSTGRES_HOST: {{ .Release.Name }}-postgresql
-  VESPA_HOST: "document-index-service"
+  VESPA_HOST: da-vespa-0.vespa-service
```

Comment:
Why is this changed? Does this have to do with the vespa charts we're hosting?

Reply:
i'm not sure the naming scheme used before ever actually worked.  This does work, but I'd like to tweak it further at some point.